src/scene/forward-renderer.js

```diff
@@ -2165,9 +2165,7 @@ pc.extend(pc, function () {
                         parameters = material.parameters;
                         for (paramName in parameters) {
                             parameter = parameters[paramName];
-                            if (!parameter.scopeId) {
-                                parameter.scopeId = device.scope.resolve(paramName);
-                            }
+                            parameter.scopeId = device.scope.resolve(paramName);

```

Comment:
So why do we need to always call resolve for every uniform now?
I believe we should just reset the default material on init, not change anything in the render loop.